.github/workflows/rust.yml

```
@@ -12,12 +12,22 @@ env:
 jobs:
   build_and_test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - nightly
+        include:
+          - rust: stable
+            all_features: "" # "alloc"
```

Comment is misleading at the moment

```suggestion
            all_features: "--features alloc" # "alloc"
```